web/packages/core/src/ruffle-player.ts

```diff
@@ -551,6 +552,9 @@ export class RufflePlayer extends HTMLElement {
             // `allowScriptAccess` can only be set in `options`.
             config.allowScriptAccess = options.allowScriptAccess;
 
+            if (config.showSwfDownload) {
```

Comment:
You could use `this.showSwfDownload = config.showSwfDownload !== false;`, similar to few lines below.

Reply:
Besides this, everything looks good to me, and I'll approve it.

Reply:
I assume you mean `this.showSwfDownload = config.showSwfDownload === true;`

Unlike with showing the context menu, this should default to false based on Mike's feedback.